internal/resources/aws/backup_vault.go

```diff
@@ -0,0 +1,163 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"fmt"
+
+	"github.com/shopspring/decimal"
+)
+
+type BackupVault struct {
+	Address                       string
+	Region                        string
+	MonthlyEFSWarmBackupGB        *float64 `infracost_usage:"monthly_efs_warm_backup_gb"`
+	MonthlyEFSColdRestoreGB       *float64 `infracost_usage:"monthly_efs_cold_restore_gb"`
+	MonthlyRDSSnapshotGB          *float64 `infracost_usage:"monthly_rds_snapshot_gb"`
+	MonthlyAuroraSnapshotGB       *float64 `infracost_usage:"monthly_aurora_snapshot_gb"`
+	MonthlyDynamodbBackupGB       *float64 `infracost_usage:"monthly_dynamodb_backup_gb"`
+	MonthlyDynamodbRestoreGB      *float64 `infracost_usage:"monthly_dynamodb_restore_gb"`
+	MonthlyFSXWindowsBackupGB     *float64 `infracost_usage:"monthly_fsx_windows_backup_gb"`
+	MonthlyFSXLustreBackupGB      *float64 `infracost_usage:"monthly_fsx_lustre_backup_gb"`
```

Comment:
```suggestion
	MonthlyFSxWindowsBackupGB     *float64 `infracost_usage:"monthly_fsx_windows_backup_gb"`
	MonthlyFSxLustreBackupGB      *float64 `infracost_usage:"monthly_fsx_lustre_backup_gb"`
```

To be consistent with how we name the `FSx...` resources

Reply:
Fixed